auslib/admin/views/forms.py

```diff
@@ -144,6 +144,7 @@ class ScheduledChangeForm(Form):
     telemetry_channel = NullableStringField("Telemetry Channel")
     telemetry_uptake = NullableStringField("Telemetry Uptake")
     when = IntegerField("When", validators=[Optional(), not_in_the_past()])
+    change_type = StringField("Change Type")
```

Comment:
Can change_type be changed when modifying an existing Scheduled Change? If not, this should only be in the forms for new Scheduled Changes.